src/Symfony/Bundle/SecurityBundle/Tests/SecurityTest.php

```diff
@@ -201,6 +201,30 @@ public function testLoginWithoutAuthenticatorThrows()
         $security->login($user);
     }
 
+    public function testLoginWithoutRequestContext()
+    {
+        $requestStack = $this->createMock(RequestStack::class);
```

Comment:
```suggestion
        $requestStack = new RequestStack();
```

(no need for the mock here)

Reply:
@xabbuh done ! thanks